tests/libcxx/expected_results.txt

```diff
@@ -1408,6 +1539,12 @@ std/ranges/range.adaptors/range.adaptor.object/range_adaptor_closure.pass.cpp:1
 # Not analyzed. Emitting easily-suppressed truncation warnings for MSVC, but also compiler errors for MSVC and Clang.
 std/algorithms/alg.modifying.operations/alg.copy/ranges.copy.pass.cpp FAIL
 
+# Not analyzed. MSVC emits "warning C4242: 'initializing': conversion from 'nasty_char_traits::int_type' to 'char', possible loss of data".

```

Comment:
Oops, I enabled test coverage using `nasty_string` for MSVC/C++20 in LLVM-135338, but forgot testing the changes with MSVC. I've just opened following-up LLVM-138375.